packages/nextjs/components/Header.tsx

```diff
@@ -94,11 +94,11 @@ export const Header = () => {
         </div>
         <Link href="/" passHref className="hidden lg:flex items-center gap-2 ml-4 mr-6 shrink-0">
           <div className="flex relative w-10 h-10">
-            <Image alt="SE2 logo" className="cursor-pointer" fill src="/logo.svg" />
+            <Image alt="DALL·E generated image" className="cursor-pointer" fill src="/logo.webp" />
```

Comment:
I think the alt should be something significative for the user who is unable to see the image to understand what it represents. Something like "Batch 13 logo"